### PR TITLE
fix backpack imbuement

### LINKED ISSUE #582: [Embuement] Capacity
image

Not giving the bonuses;

### PR DESCRIPTION
**Pull request by oalbsilva**
fix #582

### CODE REVIEW

**Comment by Bipbop69** [comment]
be the player is taking and placing the item your cap will accumulate.

**Comment by marcosvf132** [comment]
The bonus is permanent and is stacking.

**Review by jlcvp — CHANGES_REQUESTED** [review]
you need to implement onDeEquipImbueItem to remove the extra capacity added

**Comment by oalbsilva** [comment]
@dudantas  can you put the Labels "to review" and "need test"?

File: src/player.h [hunk 1]
```diff
@@ -561,13 +561,22 @@ class Player final : public Creature, public Cylinder
...
+		}
+
 		uint32_t getFreeCapacity() const {
 			if (hasFlag(PlayerFlag_CannotPickupItem)) {
 				return 0;
 			} else if (hasFlag(PlayerFlag_HasInfiniteCapacity)) {
 				return std::numeric_limits<uint32_t>::max();
 			} else {
-				return std::max<int32_t>(0, capacity - inventoryWeight);
+				return std::max<int32_t>(0, capacity + bonusCapacity - inventoryWeight);
```
Comment (gpedro) [inline src/player.h]: can we use `getCapacity()` instead `capacity + bonusCapacity`

**Comment by Bipbop69** [comment]
from what I tested now is working the bug that was occurring before is fixed

File: src/player.h (hunk 1 above)
Comment (oalbsilva) [reply src/player.h]: resolved in e18bc5f

**Review by majestyotbr — APPROVED** [review]

**Pull request merged**